index.html

```diff
@@ -0,0 +1,110 @@
+                    <span class="nav-link">
+                        Store
+                    </span>
+                </a>
+                <a href="https://web.archive.org/web/20140228043920/http://www.apple.com/mac">
+                    <span class="nav-link">
+                        Mac
+                    </span>
+                </a>
+                <a href="https://web.archive.org/web/20140228043803/http://www.apple.com/ipod/">
+                    <span class="nav-link">
+                        IPod
+                    </span>
+                </a>
+                <a href="https://web.archive.org/web/20140302053453/http://www.apple.com/iphone/">
+                    <span class="nav-link">
+                        IPhone
+                    </span>
+                </a>
+                <a href="https://web.archive.org/web/20140302053453/http://www.apple.com/ipad/">
+                    <span class="nav-link">
+                        IPad
+                    </span>
+                </a>
+                <a href="https://web.archive.org/web/20140228043510/http://www.apple.com/itunes/">
+                    <span class="nav-link">
+                        ITunes
+                    </span>
+                </a>
+                <a href="https://web.archive.org/web/20140228032616/http://www.apple.com/support/">
+                    <span class="nav-link">
+                        Support
+                    </span>
+                </a>
+                <div class="input-container">
+                    <input type="text">
+                </div>
+            </nav>
+            <div class="text-container">
+                <a href="https://web.archive.org/web/20140227031403/http://www.apple.com/your-verse">
+                   <h1>What will your verse be?</h1>
+                </a>
+                <a href="https://web.archive.org/web/20140227031403/http://www.apple.com/your-verse">
+                    <h2>
+                        Everyone has something to share. See how people are using 
+                        <br>
+                        iPad in their unique ways to add to world's story.
+                    </h2>
+                </a>
+                <div class="text-container-play">
+                    <a href="https://web.archive.org/web/20140227031403/http://www.apple.com/your-verse">
+                        <h3>Watch the "Your Verse" fillm and explore the stories </h3>
+                    </a>
+                    <a href="https://web.archive.org/web/20140227031403/http://www.apple.com/your-verse">
+                        <img src="./images/icons8-youtube-play-64.png" alt="icon">
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="modules-container-1">
+
+        </div>
+        <div class="modules-container-2">
+
+        </div>
+        <div class="modules-container-3">
+
+        </div>
+        <div class="modules-container-4">
+
+        </div>
```

Comment:
- [ ] Each of these containers should be a link (like in the [original page](https://web.archive.org/web/20140301004610/http://www.apple.com/))